sdk/textanalytics/ai-text-analytics/src/index.ts

```diff
@@ -81,8 +81,7 @@ export {
   LinkedEntity,
   Match,
   TextAnalyticsWarning,
-  SentenceAspectSentiment,
-  SentenceOpinionSentiment,
+  TokenSentimentValue,
```

Comment:
@maririos @iscai-msft -- are we making this change to the API surface?

We could do `TokenSentimentValue as SentenceAspectSentiment`, `TokenSentimentValue as SentenceOpinionSentiment`, I think if we want to keep the old names, otherwise I think this is nice.

Reply:
in .NET we don't use either of this types as we use the "generic" `TextSentiment` used in Document and Sentence sentiment.

Reply:
`TokenSentimentValue` doesn't say much about what the type actually is. It will be better to use a more descriptive name.
I leave this to JS though :)